Makefile

```diff
@@ -16,4 +16,9 @@ proto:
 generate:
 	go generate ./...
 
+.PHONY: image-push-all
```

Comment:
@ayildirim21 please review this makefile target and see if this is something we already worked on as part of the CI improvement task or something we can leverage to simplify our CI script.

Reply:
Can confrim that the existing script already implements this functionality, just need to run:

`./hack/update_examples.sh -bpe`

Will open a PR to remove this makefile target.

Reply:
please do not remove the target, instead let the target run `./hack/update_examples -bp`